event_track_ticket/tests/test_registration.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.tests.common import SavepointCase
+from odoo.exceptions import ValidationError
+
+
+# TODO: update tests
```

Comment:
to update?

Reply:
you'll have to ask the guy who wrote it :smile:.Im not sure the main difference is that the seats are handled on the track. which is checked in test_track but registrations still should work the same way.